vignettes/articles/Hector-Wider-World.Rmd

```diff
@@ -14,14 +14,14 @@ knitr::opts_chunk$set(
 
 This page is meant to serve as a reference for places where Hector has been featured. This list will be updated with time.
   
-## Hector Publications {#hector_documenation}
+## Hector Publications {#hector_documentation}
```

Comment:
This is a hyperlink used in the manual.Rmd, please also update that call in line 21